core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -3879,6 +4444,65 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
             if (cur_func->import_func_inst) {
                 wasm_interp_call_func_import(module, exec_env, cur_func,
                                              prev_frame);
+#if WASM_ENABLE_EXCE_HANDLING != 0
+                const char *uncaught_exception = wasm_get_exception(module);
+                if (uncaught_exception
+                    && strcmp(uncaught_exception,
+                              "Exception: uncaught wasm exception")
+                           == 0) {
+                    /* fix framesp */
+                    UPDATE_ALL_FROM_FRAME();
+
+                    uint32_t import_exception =
+                        0xFFFFFFFF; /* initialize imported exception index to be
+                                       invalid */
+                    /* pull external exception */
+                    uint32_t ext_exception = POP_I32();
+
+                    WASMModule *im_mod = cur_func->u.func_import->import_module;
+
+                    /* external function came back with an exception or trap */
+                    _EXCEDEBUG("exception after wasm_interp_call_func_import, "
+                               "exception_tag_index = %x, "
+                               "module->import_tag_count %d\n",
+                               ext_exception, // (int32) *( (int32*) (frame->sp
+                                              // - 1)), //exception_tag_index,
+                               module->module->import_tag_count);
+
+                    /* lookup exception in import tags */
+                    for (uint32_t import_tag_index = 0;
```

Comment:
Had better declare `uint32 import_tag_index` above to keep C coding style.

Reply:
updated to accordingly.

Reply:
Updated to keep coding style.